components/derive/src/service_factory.rs

```diff
@@ -120,20 +120,20 @@ impl ToTokens for ServiceFactory {
         let (impl_generics, ty_generics, where_clause) = self.generics.split_for_impl();
 
         let expanded = quote! {
-            impl #impl_generics #cr::runtime::rust::ServiceFactory for #name #ty_generics #where_clause {
-                fn artifact_id(&self) -> #cr::runtime::ArtifactId {
-                    #cr::runtime::ArtifactId {
-                        runtime_id: #cr::runtime::RuntimeIdentifier::Rust as _,
+            impl #impl_generics #cr::ServiceFactory for #name #ty_generics #where_clause {
+                fn artifact_id(&self) -> exonum::runtime::ArtifactId {
```

Comment:
You've changed the types to local for the `ServiceDispatcher` derive macro, but not here. Is this intentional?